Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -167,6 +167,13 @@ def subnext(self):
                                    '--datatier':'NANOAOD',
                                    '--eventcontent':'NANOAOD'}])
 
+steps['l1DPGNANO_data13.0']=merge([{'-s' : 'RAW2DIGI,NANO:@L1DPG',
```

Comment:
can you please do something like `muPOGNANO_data13.0` (merge with `steps['NANO_data13.0']`) instead ? I could ask that you modify `muDPGNANO_data13.0` and `muDPGNANOBkg_data13.0` the same way to have it consistent, but this is not necessary here